### PR TITLE
Dcsobral/ch11107/test pr creation

### PR DESCRIPTION
**Pull request by dcsobral**

### CODE REVIEW

File: core/src/main/scala/precog/SbtPrecogBase.scala [hunk 1]
```diff
@@ -348,10 +355,12 @@ abstract class SbtPrecogBase extends AutoPlugin {
       }.toString,
       trickleGitConfig := {
         import sbttrickle.git.GitConfig
-        (sys.env.get("GITHUB_ACTOR"), sys.env.get("GITHUB_TOKEN")) match {
+        val baseConf = (sys.env.get("GITHUB_ACTOR"), sys.env.get("GITHUB_TOKEN")) match {
```
Comment (djspiewak) [inline core/src/main/scala/precog/SbtPrecogBase.scala]: Do we still need `GITHUB_ACTOR`?

**Comment by dcsobral** [comment]
I guess we could use underscore instead, but the API on trickle is not
github-specific, so it still requires a user. Github workflow is still
setting it.

-- 
Daniel C. Sobral

File: core/src/main/scala/precog/SbtPrecogBase.scala (hunk 1 above)
Comment (dcsobral) [reply core/src/main/scala/precog/SbtPrecogBase.scala]: I guess we could use underscore instead, but the API on trickle is not
github-specific, so it still requires a user. Github workflow is still
setting it.

**Comment by djspiewak** [comment]
> Github workflow is still setting it.

Only because trickle still demands it.

File: core/src/main/scala/precog/Changes.scala [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package precog
+
+class Changes {
+
+}
```
Comment (djspiewak) [inline core/src/main/scala/precog/Changes.scala]: Stray file that was never compiled and thus doesn't have headers.

**Review by djspiewak — CHANGES_REQUESTED** [review]

File: core/src/main/scala/precog/Changes.scala (hunk 2 above)
Comment (dcsobral) [reply core/src/main/scala/precog/Changes.scala]: It wasn't supposed to be added. It sneaked in when I made the GITHUB_ACTOR change.

**Pull request merged**